azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/configuration/CallCompositeConfiguration.kt

```diff
@@ -41,8 +42,9 @@ internal class CallCompositeConfiguration {
         // Gets a config by it's ID
         // May return null if the Configuration becomes garbage collected
         fun getConfig(id: Int): CallCompositeConfiguration = configs[id]
-            ?: throw RuntimeException(
-                "This ID is not valid, and no entry exists in the map. Please file a bug, this is an error in the composite"
+            ?: throw CallCompositeException(
+                "This ID is not valid, and no entry exists in the map. Please file a bug, this is an error in the composite",
+                NullPointerException()
```

Comment:
I do not think that `NullPointerException` is the best here. It is more suited for the variable that happened to be not initialized. Here we have `key` not found in the map. There should not be a valid scenario when key is not preset. Should we consider `IllegalStateException` for this?